src/core/services/google-maps-types.ts

```diff
@@ -245,7 +245,7 @@ export interface PolylineOptions {
   draggable?: boolean;
   editable?: boolean;
   geodesic?: boolean;
-  icon?: Array<IconSequence>;
+  icons?: Array<IconSequence>;
```

Comment:
IconSequence should be exported here: https://github.com/SebastianM/angular-google-maps/blob/master/src/core/map-types.ts